README.md

```diff
@@ -96,7 +96,47 @@ not be repeated, and should be placed in a folder named by date.
 Repeated events are always shown at the top of the events page, and 
 do not expire.
 
-### 4. How do I add a page redirect?
+### 4. How do I add a community document?
+
+A community document is a living document that represents a community effort
+to discuss a question or idea. We call it "living" because we encourage community
+members to edit and contribute to the documents over time, and so there isn't
+a "publish" data associated with it. For this reason, we use [the wiki](https://github.com/USRSE/usrse.github.io/wiki) associated
+with the repository here. 
+
+![assets/img/usrse-book-small.png](assets/img/usrse-book-small.png)
+
+If you ever need to clone this content, you can do:
+
+```bash
+$ git clone git@github.com/USRSE/usrse.github.io/wiki
+```
+
+_NOTE_: During the testing phase of the living documents, the wiki will only be editable by community (repository)
+members, however we might open this up to include others as well. 
```

Comment:
Maybe change this to:

_NOTE_: During the testing phase of the living documents, we currently plan for the wiki to only be editable by community (repository)
 members.

Reply:
All set!